clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinator.java

```diff
@@ -729,6 +729,23 @@ public void handle(OffsetCommitResponse commitResponse, RequestFuture<Void> futu
     private class OffsetFetchResponseHandler extends CoordinatorResponseHandler<OffsetFetchResponse, Map<TopicPartition, OffsetAndMetadata>> {
         @Override
         public void handle(OffsetFetchResponse response, RequestFuture<Map<TopicPartition, OffsetAndMetadata>> future) {
+            if (response.hasError()) {
+                Errors error = Errors.forCode(response.errorCode());
```

Comment:
I'll add a method to `OffsetFetchResponse` that returns the actual `Errors` value.